### PR TITLE
fix logits vocab

### PR DESCRIPTION
**Pull request by minhthuc2502**

### CODE REVIEW

**Comment by koren-v** [comment]
@minhthuc2502, thanks for this fix!

What about sorting logits in [sort_hypotheses](https://github.com/OpenNMT/CTranslate2/blob/master/src/decoding.cc#L206)?

**Comment by minhthuc2502** [comment]
> @minhthuc2502, thanks for this fix!
> 
> What about sorting logits in [sort_hypotheses](https://github.com/OpenNMT/CTranslate2/blob/master/src/decoding.cc#L206)?

Thank you for your remark.
